feat(nats): add doc for NATS product (#4764)
scaleway/docs-content: pages/nats/how-to/monitor-nats-cockpit.mdx

1. Generate your Grafana credentials
2. Access the product's metrics dashboard

These steps are described below.

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- At least one [NATS account](/nats/how-to/create-account/)

## How to generate Grafana credentials

To view your metrics with Cockpit, you will use a Grafana dashboard which is accessible from the Scaleway console. First, you must create a user and their associated credentials (a username and password) for Grafana. This can be done directly from the Scaleway console, [by following this procedure](/cockpit/how-to/retrieve-grafana-credentials/).

## How to access your NATS dashboard and view your metrics

1. Click **NATS** in the **Serverless** section of the Scaleway console side menu.

> 2. Click the **protocol** you want to view metrics for, and navigate to the **Metrics** tab.

Author: ```suggestion
2. Click the **Metrics** tab.
```

3. Click **Open Grafana metrics dashboard**.

4. Enter your Grafana username and password to log in.

>     You are directed to the dashboard for the product in question, where you can view all your metrics.

Author: ```suggestion
    You are directed to the NATS dashboard, where you can view all your metrics.
```
